Mirror/Weaver/Weaver.cs

```diff
@@ -1444,7 +1444,7 @@ static void SetupTargetTypes()
 
             gameObjectInequality = ResolveMethod(unityObjectType, "op_Inequality");
 
-            UBehaviourIsServer  = ResolveMethod(NetworkBehaviourType, "get_isServer");
+            UBehaviourIsServer = ResolveMethod(NetworkBehaviourType, "get_isServer");
```

Comment:
Can you remove the unnecessary whitespaces please?

Reply:
The whitespace was removed in this commit (automatic formatting, I figured I'd leave that in)

Reply:
Can you still remove it for this pull request and then maybe add another one just for that later?
keeping it simple and stupid :)

Reply:
Sure, here you go